e2e2/internal/framework_extensions/helm.go

```diff
@@ -0,0 +1,65 @@
+package frameworkext
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chart/loader"
+	"helm.sh/helm/v3/pkg/cli"
+	"helm.sh/helm/v3/pkg/getter"
+	"helm.sh/helm/v3/pkg/release"
+	"helm.sh/helm/v3/pkg/repo"
+)
+
+func InstallPackagedRelease(repoName string, repoUrl string, chart string, releaseName string, namespace string,
+	values map[string]interface{}) (*release.Release, error) {
+	entry := &repo.Entry{
+		Name:                  repoName,
+		URL:                   repoUrl,
+		InsecureSkipTLSverify: true,
+	}
+
+	setting := cli.New()
+	r, err := repo.NewChartRepository(entry, getter.All(setting))
+	_, err = r.DownloadIndexFile()
+
+	file := repo.NewFile()
+	file.Update(entry)
+
+	actionConfig := new(action.Configuration)
+	actionConfig.Init(setting.RESTClientGetter(), namespace, os.Getenv("HELM_DRIVER"), func(format string, v ...interface{}) {
+		message := fmt.Sprintf(format, v...)
+		log.Println(message)
+	})
+
+	client := action.NewInstall(actionConfig)
+	client.ReleaseName = releaseName
+	client.Namespace = namespace
+	cp, err := client.ChartPathOptions.LocateChart(chart, setting)
+	chartReq, err := loader.Load(cp)
+	release, err := client.Run(chartReq, values)
+	return release, err
+}
+
+func Label(resource string, label string,
+	namespace string, kubeconfig string) ([]byte, error) {
+	cmd := exec.Command("kubectl", "label", "--overwrite", resource,
+		label, "--namespace", namespace, "--kubeconfig", kubeconfig)
+	return cmd.CombinedOutput()
+}
+
+func ResourceExists(resource string, namespace string, kubeconfig string) ([]byte, error) {
+	cmd := exec.Command("kubectl", "get", resource, "--namespace", namespace,
```

Comment:
Same here, the e2e-framework has utilities for this

Reply:
Removed this function for now as I'm deleting the resource instead of patching for labels/annotations